[INFO   ] Logger: Record log in C:\Users\aluno.sesipaulista\.kivy\logs\kivy_24-05-03_4.txt
[INFO   ] deps: Successfully imported "kivy_deps.angle" 0.4.0
[INFO   ] deps: Successfully imported "kivy_deps.glew" 0.3.1
[INFO   ] deps: Successfully imported "kivy_deps.sdl2" 0.7.0
[INFO   ] Kivy: v2.3.0
[INFO   ] Kivy: Installed at "C:\Users\aluno.sesipaulista\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\kivy\__init__.py"
[INFO   ] Python: v3.12.3 (tags/v3.12.3:f6650f9, Apr  9 2024, 14:05:25) [MSC v.1938 64 bit (AMD64)]
[INFO   ] Python: Interpreter at "C:\Users\aluno.sesipaulista\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\python.exe"
[INFO   ] Logger: Purge log fired. Processing...
[INFO   ] Logger: Purge finished!
[INFO   ] Factory: 195 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2 (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL" graphics system
[INFO   ] GL: GLEW initialization succeeded
[INFO   ] GL: Backend used <glew>
[INFO   ] GL: OpenGL version <b'4.6.0 - Build 31.0.101.4502'>
[INFO   ] GL: OpenGL vendor <b'Intel'>
[INFO   ] GL: OpenGL renderer <b'Intel(R) Iris(R) Xe Graphics'>
[INFO   ] GL: OpenGL parsed version: 4, 6
[INFO   ] GL: Shading version <b'4.60 - Build 31.0.101.4502'>
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <32>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "kivy\\_event.pyx", line 235, in kivy._event.EventDispatcher.__init__
[WARNING] stderr: TypeError: object.__init__() takes exactly one argument (the instance to initialize)
[WARNING] stderr: 
[WARNING] stderr: The above exception was the direct cause of the following exception:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "c:\Users\aluno.sesipaulista\Desktop\lou\oiooo.py\30.py", line 8, in <module>
[WARNING] stderr:     MinhaApp().run()
[WARNING] stderr:   File "C:\Users\aluno.sesipaulista\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\kivy\app.py", line 955, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "C:\Users\aluno.sesipaulista\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\kivy\app.py", line 925, in _run_prepare
[WARNING] stderr:     root = self.build()
[WARNING] stderr:            ^^^^^^^^^^^^
[WARNING] stderr:   File "c:\Users\aluno.sesipaulista\Desktop\lou\oiooo.py\30.py", line 5, in build
[WARNING] stderr:     return ToggleButton(Text='louyse linda', state='normal')
[WARNING] stderr:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "C:\Users\aluno.sesipaulista\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\kivy\uix\behaviors\togglebutton.py", line 84, in __init__
[WARNING] stderr:     super(ToggleButtonBehavior, self).__init__(**kwargs)
[WARNING] stderr:   File "C:\Users\aluno.sesipaulista\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\kivy\uix\behaviors\button.py", line 121, in __init__
[WARNING] stderr:     super(ButtonBehavior, self).__init__(**kwargs)
[WARNING] stderr:   File "C:\Users\aluno.sesipaulista\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\kivy\uix\label.py", line 319, in __init__
[WARNING] stderr:     super(Label, self).__init__(**kwargs)
[WARNING] stderr:   File "C:\Users\aluno.sesipaulista\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.12_qbz5n2kfra8p0\LocalCache\local-packages\Python312\site-packages\kivy\uix\widget.py", line 357, in __init__
[WARNING] stderr:     super(Widget, self).__init__(**kwargs)
[WARNING] stderr:   File "kivy\\_event.pyx", line 238, in kivy._event.EventDispatcher.__init__
[WARNING] stderr: TypeError: Properties ['Text'] passed to __init__ may not be existing property names. Valid properties are ['allow_no_selection', 'always_release', 'anchors', 'background_color', 'background_disabled_down', 'background_disabled_normal', 'background_down', 'background_normal', 'base_direction', 'bold', 'border', 'center', 'center_x', 'center_y', 'children', 'cls', 'color', 'disabled', 'disabled_color', 'disabled_outline_color', 'ellipsis_options', 'font_blended', 'font_context', 'font_direction', 'font_family', 'font_features', 'font_hinting', 'font_kerning', 'font_name', 'font_script_name', 'font_size', 'group', 'halign', 'height', 'ids', 'is_shortened', 'italic', 'last_touch', 'limit_render_to_text_bbox', 'line_height', 'markup', 'max_lines', 'min_state_time', 'mipmap', 'motion_filter', 'opacity', 'outline_color', 'outline_width', 'padding', 'padding_x', 'padding_y', 'parent', 'pos', 'pos_hint', 'refs', 'right', 'shorten', 'shorten_from', 'size', 'size_hint', 'size_hint_max', 'size_hint_max_x', 'size_hint_max_y', 'size_hint_min', 'size_hint_min_x', 'size_hint_min_y', 'size_hint_x', 'size_hint_y', 'split_str', 'state', 'strikethrough', 'strip', 'text', 'text_language', 'text_size', 'texture', 'texture_size', 'top', 'underline', 'unicode_errors', 'valign', 'width', 'x', 'y']
